app/views/users/registrations/new.html.erb

```diff
@@ -17,7 +17,7 @@
           <div class="space-y-6">
             <div>
               <%= form.label :username %>
-              <%= form.text_field :username, required: true, autofocus: params[:email].present?, class: 'dark-form-input', data: { test_id: 'username_field', attr: 'form-validation.username', action: 'blur->form-validation#validate' } %>
+              <%= form.text_field :username, required: true, autofocus: true, class: 'dark-form-input', data: { test_id: 'username_field', attr: 'form-validation.username', action: 'blur->form-validation#validate' } %>
```

Comment:
Hey @rudnam, it looks like our client side validation, we're being a bit to agressive triggering validations on blur. We can change this to on input to get the tests to pass for now.

```suggestion
              <%= form.text_field :username, required: true, autofocus: true, class: 'dark-form-input', data: { test_id: 'username_field', attr: 'form-validation.username', action: 'input->form-validation#validate' } %>
```